charts/stable/multus/values.yaml

```diff
@@ -6,57 +6,71 @@
 #
 
 image:
+  # -- image repostory
   repository: ghcr.io/k8snetworkplumbingwg/multus-cni
+  # -- image tag
   tag: v3.7.1
+  # -- image pull policy
   pullPolicy: IfNotPresent
 
-controllerType: daemonset
+controller:
+  # -- deploy to all nodes
+  type: daemonset
```

Comment:
Would it make sense to put this as a "hardcoded" value?

Reply:
jaja, I see you like the hardcoded values - yes, this can be moved there :-)

Reply:
haha, well, I do like them in case it makes no sense for the chart user to set them in values.yaml 😉